### PR TITLE
Mobile first work about

### PR DESCRIPTION
**Pull request by agneta1991**
-added work and about sections
-fixed lint errors
-hover was added to all the buttons and to a 'languages' section 
-previous project issues that were hurting the current design were fixed.

### CODE REVIEW

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @agneta1991 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

It seems that there are some conflicts in your project that need to be resolved before the pull request can be successfully reviewed. It's important to address these conflicts to ensure the smooth integration of your changes into the existing codebase.

To resolve the conflicts, you can follow these steps:

1. Pull the latest changes from the main branch into your local branch.
2. Open the conflicting files in your code editor and locate the areas marked with conflict markers (<<<<<<<, =======, >>>>>>>).
3. Review the conflicting code and make the necessary changes to resolve the conflicts. You may need to choose between different code versions or manually merge the changes.
4. Save the modified files after resolving the conflicts.
5. Test your changes to ensure they work correctly and do not introduce any new issues.
6. Once you have resolved all conflicts, push the changes to your remote branch.
 
After resolving the conflicts and pushing the changes, the GitHub linter should provide a green checkmark indicating that the linter passes.

Take your time to carefully resolve the conflicts, ensuring that your changes integrate smoothly with the existing code. If you need further assistance, feel free to ask for help.

To learn more check [this document](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -5,44 +5,223 @@
...
+                    </div>
+                    <button class="see-project"> See Project</button>
+                </div>
+            </div>
+        </section>
+    </section>
+    <section class="about-myself second-part">
+        <section class="headline">
+            <h1 class="greeting">About Myself
+            </h1>
```
Comment (Ofarouq310) [inline index.html:170]: - [ ] The whole `about` section as stated in the requirements had to be laid out using `flexbox`, kindly implement that.

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @agneta1991, ✋ 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

giphy

## Highlights
✅ Descriptive PR title.
✅ Followed GitHub workflow.
✅ No linters errors.

## Required Changes ♻️

- [ ] As for the PR description, you have to summarize what you implemented in the project using a bullet list, a checklist, or a numbers list, in your case it could be something like:
    - Added a header section to the portfolio,
    - Added a headline section with an introduction paragraph.
    - etc...
     
or anything that has the same intent of explaining the new additions to your project.

- Your Readme file needs some attention in a couple of sections: 

  - [x] The `Contributing` section is missing the `issues` link, kindly fix it. Refer to the main [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for a closer look.
  
  - [x] In The `Show Your Support` section, you should add something like `If you like this project, give me a star`, kindly fix it since what you wrote seems to be a description of your project, it seems inappropriate. Refer to the main [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for a closer look.


_Check the comments under the review._

## Optional suggestions

 - Concerning commit messages, kindly make sure they should be short, concise, descriptive of an action, written in the imperative tense, and capitalized, take a closer look in the next picture for some great descriptive examples.
image

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -5,44 +5,223 @@
...
+                            <img src="images/Vector (2).svg" alt="social media">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="images/Vector (3).svg" alt="social media">
+                        </a>
+                    </li>
+                </ul>
+            </section>
```
[the remark this answers is not in the document]
Comment (agneta1991) [reply index.html:65]: @shahadat3669 as i resolved the issues of my previous project in my current branch, now my pull request says there is a conflict. how can i resolve it without merging branches?

**Comment by agneta1991** [comment]
@shahadat3669 regarding PR description, i think i have mixed it up and left my description in the comment rather than description section, could you advise me how i could edit PR description?

**Review by divinecharlotte — APPROVED** [review]
# Hello @agneta1991     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Good design implementation👍 
- Linters are passing👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**